includes/Utilities/BackgroundJobCleanup.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Utilities;
+
+use SkyVerge\WooCommerce\Facebook\Heartbeat;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class BackgroundJobCleanup
+ *
+ * Responsible for cleaning up old completed background sync jobs from SkyVerge background job system.
+ * Each job is represented by a row in wp_options table, and these can accumulate over time.
+ *
+ * Note - this is closely coupled to the SkyVerge background job system, and is essentially a patch to improve it.
+ *
+ * @see SV_WP_Background_Job_Handler
+ *
+ * @since x.x.x
+ */
+class BackgroundJobCleanup {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->add_hooks();
+	}
+
+	/**
+	 * Add hooks.
+	 */
+	public function add_hooks() {
+		// Register our cleanup routine to run on the hourly heartbeat.
+		add_action( Heartbeat::HOURLY, array( $this, 'clean_up_old_completed_options' ) );
```

Comment:
Could add a `DAILY` heartbeat for things like this.

Reply:
If we increase the `limit` in the `delete` query I'd suggest we increase this to _every 12 hours_ or even _daily_.

Reply:
Will need to extend `Heartbeat` to support that, but I agree it's worth doing.

Reply:
Keen for @danielbitzer feedback on this :)

Reply:
Added a `daily` heartbeat and switched this to using it. Now .. we need a day+ to test!! @danielbitzer can you review this? If review all good, I'm thinking we merge this (since it's low risk) and then we can test it more on master to ensure it's cleaning up.

Reply:
I like the idea of adding doing this daily 💯